presto-main/src/test/java/com/facebook/presto/sql/planner/TestLogicalPlanner.java

```diff
@@ -384,6 +384,48 @@ public void testQuantifiedComparisonGreaterEqualAny()
         assertEqualPlans(plan(query), plan(rewrittenSql));
     }
 
+    @Test
+    public void testQuantifiedComparisonEqualAll()
+    {
+        String query = "SELECT orderkey, custkey FROM orders WHERE orderkey = ALL (VALUES CAST(5 as BIGINT), CAST(3 as BIGINT))";
+
+        assertPlan(query, LogicalPlanner.Stage.CREATED,
+                anyTree(
+                        project(
+                                filter("min = max AND orderkey = min",
```

Comment:
`min`, `max` and `orderkey` are just aliases here. They have nothing the plan symbols. In order to check their origin you would need to write something like that.

```
        assertPlan(query, LogicalPlanner.Stage.CREATED,
                anyTree(
                        project(
                                filter("NOT (MIN = MAX AND O = MIN)",
                                        apply(ImmutableList.of(),
                                                project(
                                                        tableScan("orders").withSymbol("orderkey", "O"),
                                                node(EnforceSingleRowNode.class,
                                                        node(AggregationNode.class,
                                                                project(
                                                                        project(
                                                                                node(ValuesNode.class)))).withSymbol("min", "MIN").withSymbol("max", "MAX") // withSymbols methods should be called for aggregationNode
)
                                        )))));
```